src/notes.ts

```diff
@@ -154,10 +157,12 @@ chrome.storage.onChanged.addListener((changes, areaName) => {
     }
 
     if (changes["notes"]) {
-      const oldNotes = changes["notes"].oldValue;
-      const newNotes = changes["notes"].newValue;
+      const oldNotes: NotesObject = changes["notes"].oldValue;
+      const newNotes: NotesObject = changes["notes"].newValue;
       state.notes = newNotes;
 
+      state.clipboard = changes["clipboard"] ? changes["clipboard"].newValue : state.clipboard;
```

Comment:
If there was no clipboard note set before using it, it is set automatically to be "Clipboard" in the background.